commerce_coordinator/apps/ecommerce/views.py

```diff
@@ -57,3 +59,39 @@ def get(self, request):
         )
 
         return Response(format_signal_results(results))
+
+
+class OrderView(APIView):
```

Comment:
@HammadAhmadWaqas Sorry, I know you haven't submitted this PR for review yet...

What do you think of changing this to `CreateOrderView` or something similar? I realized I was the one who originally called this `OrderView`, but that was before we changed this to a GET request. 😅